exercise/templates/exercise/_user_results.html

```diff
@@ -44,7 +51,7 @@ <h3 class="panel-title">
       {{ module.name|parse_localization }}
     </h3>
   </a>
-  <div class="collapse{% if open and accessible %} in{% endif %}" id="module{{ module.id }}">
```

Comment:
Previously, when a module got closed (after the close time), the div was also closed/collapsed. After your change, the div is always open forever after the open time. This should be fixed. The div should be closed after the module close time. This probably affects the `a.panel-heading` element in the top too.